src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -163,6 +164,49 @@ public void canCloseAnOpenLoanByCheckingInTheItem() {
     verifyCheckInOperationRecorded(nod.getId(), checkInServicePointId);
   }
 
+@Test
+public void verifyItemEffectiveLocationIdAtCheckOut() {
+
+  final IndividualResource james = usersFixture.james();
+
+  final UUID checkInServicePointId = servicePointsFixture.cd1().getId();
+
+  final IndividualResource homeLocation = locationsFixture.basedUponExampleLocation(
+    item -> item.withPrimaryServicePoint(checkInServicePointId));
+
+  final String anotherLocationId = locationsFixture.thirdFloor().getId().toString();
+
+  final IndividualResource nod = itemsFixture.basedUponNod(
+    item -> item
+      .withTemporaryLocation(homeLocation.getId())
+      .withEnumeration("v.70:no.1-6")
+      .withChronology("1987:Jan.-June")
+      .withVolume("testVolume"));
+
+  checkOutFixture.checkOutByBarcode(nod, james, new DateTime(2018, 3, 1, 13, 25, 46, DateTimeZone.UTC));
```

Comment:
```suggestion
  checkOutFixture.checkOutByBarcode(nod, james);
```

Data is not required here, it is mostly used for some tests that verifies loan renewal, etc.